std/array.d

```diff
@@ -295,6 +295,19 @@ if (is(Range == U*, U) && isIterable!U && !isAutodecodableString!Range && !isInf
     R().array;
 }
 
+// Test that `array(scope InputRange r)` returns a non-scope array
+// https://issues.dlang.org/show_bug.cgi?id=23300
+@safe pure nothrow unittest
+{
+    @safe int[] fun()
+    {
+        import std.algorithm : map;
```

Comment:
```
grep -nr 'import std.algorithm : ' $(find etc std -name '*.d') ; test $? -eq 1
std/array.d:304:        import std.algorithm : map;
```
not sure what the style complaint is about here but it is complaining.

Reply:
It wants me to specify the subpackage `std.algorithm.iteration`